packages/manager/modules/error-page/src/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,10 @@
+{
+  "manager_error_page_title": "Oops …!",
+  "manager_error_page_button_cancel": "Annuler",
+  "manager_error_page_button_action_pay_debt": "Régler ma dette",
```

Comment:
Are you sure that both `manager_error_page_button_action_pay_debt` and `manager_error_page_button_action_contact_support` are related to this module?